src/utils/url.js

```diff
@@ -154,3 +154,131 @@ export function stringifyQuery(query) {
 
     return search
 }
+
+/* Import URL utils start */
+
+const ADMIN_URL_REGEXP = new RegExp(
```

Comment:
`ADMIN` name is a bit confusing with administrator url, maybe use `BGDI_URL_REGEXP` so it is clearer that this regex is for the BGDI projects urls.

Reply:
removed as proxy no need anymore